topics/ecology/tutorials/remote-sensing/tutorial.md

```diff
@@ -0,0 +1,444 @@
+---
+layout: tutorial_hands_on
+
+title: Sentinel 2 biodiversity
+zenodo_link: ''
+questions:
+- How to get spectral and biodiversity indicators from remote sensing data ?
+- Which kind of ecosystem are you studying ? global or canopy data ?
+objectives:
+- Getting Sentinel 2 data and reformatting them in a generalized way
+- Computing spectral indices such as the NDVI
+- Calculating and visualizing biodiversity indicators
+- Comparing with in-situ data
+time_estimation: 48H
+key_points:
```

Comment:
Last request: Key points should be conclusions from the tutorial that are important to remember. Think about what the lessons are you want learners to have learned from this tutorial. So can you rephrase them to be full sentences? Things like "It is always important to do X because ..", "Always keep in mind X when interpreting Y", "X is a method to do Y", etc.